GOVERNANCE.md

```diff
@@ -0,0 +1,127 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release, Long Term Support (LTS) and
+Canary in the Gold Mine (CitGM) teams. It manages the release
+and long term support schedule and policies for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
+* Schedule for all releases.
+* Contribution policy for the Release repository.
+* Conduct guidelines for the Working group.
+* Administration of Long Term Support policy for all releases.
+
+For the current list of WG members, see the project README.md.
+
+## Collaborators
+
+The Release GitHub repository is maintained by the WG (all WG
+members are Collaborators for the Release repository) and additional
+Collaborators who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These individuals
```

Comment:
It is the LTS repo to be remained to the Release repo.  I'll change from project to Release repository.